src/cpu.c

```diff
@@ -632,8 +642,9 @@ static int cpu_read(void) {
   FILE *fh;
   char buf[1024];
 
-  char *fields[9];
+  char *fields[11];
   int numfields;
+  long long user_value, nice_value, value;
```

Comment:
Hi!

In current codestyle variable should be declared when it used at first time, if that is possible.

Thanks for your work on this!

Reply:
Ok, adjusted :)